spec/directives/import/load.hrx

```diff
@@ -0,0 +1,185 @@
+<===> explicit_extension/sass/input.scss
+@import "other.sass"
+
+<===> explicit_extension/sass/other.sass
+a
+  syntax: sass
+
+<===> explicit_extension/sass/other.scss
+a {syntax: scss}
+
+<===> explicit_extension/sass/other.css
+a {syntax: css}
+
+<===> explicit_extension/sass/output.css
+a {
+  syntax: sass;
+}
+
+<===>
+================================================================================
+<===> explicit_extension/scss/input.scss
+@import "other.scss"
+
+<===> explicit_extension/scss/other.sass
+a
+  syntax: sass
+
+<===> explicit_extension/scss/other.scss
+a {syntax: scss}
+
+<===> explicit_extension/scss/other.css
+a {syntax: css}
+
+<===> explicit_extension/scss/output.css
+a {
+  syntax: scss;
+}
+
+<===>
+================================================================================
+<===> precedence/scss_before_css/options.yml
+---
+:todo:
+- libsass # sass/libsass#2699
+
+<===> precedence/scss_before_css/input.scss
+@import "other";
+
+<===> precedence/scss_before_css/other.scss
+a {syntax: scss}
+
+<===> precedence/scss_before_css/other.css
+a {syntax: css}
+
+<===> precedence/scss_before_css/output.css
+a {
+  syntax: scss;
+}
+
+<===>
+================================================================================
+<===> precedence/sass_before_css/options.yml
+---
+:todo:
+- libsass # sass/libsass#2699
+
+<===> precedence/sass_before_css/input.scss
+@import "other";
```

Comment:
Probably not worth it or covered elsewhere, but I was wondering if it'd be worth testing the error case where both ".sass" and ".scss" are present (since it tests the other extension combos: css+sass and css+scss).

Reply:
Errors are already covered [here](https://github.com/sass/sass-spec/blob/master/spec/directives/import/error/conflict.hrx).